tests/cypress/page-object/workflow.page.ts

```diff
@@ -0,0 +1,28 @@
+import { BasePage } from './base.page'
+
+class WorkflowPage extends BasePage {
+    public static readonly SITE = 'digitall'
+    public static readonly REVIEWER_USERNAME = 'ace'
+    public static readonly REVIEWER_PASSWORD = 'ventura'
+
+    goTo(user?: authMethod) {
+        cy.goTo('/jahia/workflow', user)
+        return cy
+    }
+
+    openWorkflowAndVerifyButtons() {
```

Comment:
method name should emphasis it is for the reviewer role (it won't work for the reschedule step for example)